### PR TITLE
update quarto-python agenda

### PR DESCRIPTION
**Pull request by hamelsmu**
cc: @jjallaire @mine-cetinkaya-rundel

### CODE REVIEW

File: workshops/quarto-python/index.qmd [hunk 1]
```diff
@@ -11,45 +11,45 @@ languages:
...
+
+Enter Quarto - an innovative, open-source scientific and technical publishing system compatible with Jupyter Notebooks and other popular mediums. Quarto provides data scientists with a seamless way to publish their work in a high-quality format that is easily accessible and shareable. With Quarto, researchers can turn their Jupyter Notebooks into professional-looking publications in a variety of formats, including web pages, books, and slides.
+
+In this talk, we will demonstrate how Quarto enables data scientists to turn their work products into professional, high-quality publications, websites, blog posts, and other shareable artifacts.  As a bonus, we will also discuss how you can create and document Python packages using Jupyter notebooks and Quarto with the help of [nbdev](https://nbdev.fast.ai/).
```
Comment (mine-cetinkaya-rundel) [inline workshops/quarto-python/index.qmd:23]:
```suggestion
In this workshop, we will demonstrate how Quarto enables data scientists to turn their work products into professional, high-quality publications, websites, blog posts, and other shareable artifacts.  As a bonus, we will also discuss how you can create and document Python packages using Jupyter notebooks and Quarto with the help of [nbdev](https://nbdev.fast.ai/).
```

**Comment by mine-cetinkaya-rundel** [comment]
This looks great, thank you @hamelsmu! Would you also add a brief bio at the end in the instructor section?

I'll be adding the photos later, so just leaving the photo there is perfect, thanks!

**Comment by hamelsmu** [comment]
@mine-cetinkaya-rundel ok I added my bio!

**Comment by mine-cetinkaya-rundel** [comment]
Thank you!

**Pull request merged**